views/renderWeather.js

```diff
@@ -0,0 +1,53 @@
+class renderWeather {
+    _data;
+    render(weather) {
```

Comment:
My initial concept was using one parent class and then make child classes from it but it didn't work out due to lack of functionality. (As we discussed on the call)  I kept this _data as it might needed in future. But for now I have removed it.